pkg/jira/jira.go

```diff
@@ -86,7 +87,7 @@ func (c *Verifier) ghUnlabeledPRs(extPR pr) ([]pr, error) {
 
 func (c *Verifier) verifyExtPRs(issue *jiraBaseClient.Issue, extPRs []pr, errs []error, tagName string) (bool, string, []error, bool) {
 	var success bool
-	message := fmt.Sprintf("Jira Issue-fix included in accepted release %s", tagName)
+	message := fmt.Sprintf("Bugfix included in accepted release %s", tagName)
```

Comment:
It's weird when referring to Jira Issues as Bugs... Could we just call it a "Fix"?  Maybe?